app/routes/news._index.mdx

```diff
@@ -26,6 +26,24 @@ import { Anchor } from '~/components/Anchor'
 
 ## 2024
 
+    <CollectionItem
+      className="maxw-none"
+      variantComponent={
+        <CollectionCalendarDate datetime={'July 29, 2024'} />
+      }
+    >
+      <CollectionHeading headingLevel="h3">
+        <Anchor>Circulars are Now Available via Kafka, and New Schema Release v4.1.0</Anchor>
+      </CollectionHeading>
+      <CollectionDescription>
+        - **Circulars can now be streamed over Kafka in JSON format** on the topic `gcn.circulars`.  They can be added to your existing Kafka streaming codes or generated via the [QuickStart interface](/quickstart).
+        - **Schema v4.1.0** includes the following updates:
+          - Inclusion of [Circulars schema](https://gcn.nasa.gov/docs/schema/v4.1.0/gcn/circulars.schema.json)
```

Comment:
Rather than "Inclusion of" and "Modification of" how about "Include" and "Modify" so that these resemble complete sentences in the imperative mood?